react/src/components/ImageList.tsx

```diff
@@ -0,0 +1,414 @@
+              tag={row?.tag}
+              labels={row?.labels as { key: string; value: string }[]}
+            />
+          ) : null}
+        </CellWrapper>
+      ),
+    },
+    {
+      title: t('environment.Digest'),
+      dataIndex: 'digest',
+      key: 'digest',
+      sorter: (a, b) =>
+        a?.digest && b?.digest ? a.digest.localeCompare(b.digest) : 0,
+      render: (text, row) => (
+        <CellWrapper>
+          <Typography.Text ellipsis> {row.digest}</Typography.Text>
+        </CellWrapper>
+      ),
+    },
+    {
+      title: t('environment.ResourceLimit'),
+      dataIndex: 'resource_limits',
+      key: 'resource_limits',
+      render: (text, row) =>
+        row?.resource_limits?.map((resource_limit) => (
+          <ResourceNumber
+            type={resource_limit?.key || ''}
+            value={resource_limit?.min || '0'}
+            max={resource_limit?.max || ''}
+          />
+        )),
+    },
+    {
+      title: t('general.Control'),
+      key: 'control',
+      dataIndex: 'control',
+      fixed: 'right',
+      render: (text, row) => (
+        <CellWrapper>
+          <Flex direction="row" align="stretch" justify="center" gap="xxs">
+            <Button
+              type="text"
+              icon={
+                <SettingOutlined
+                  style={{
+                    color: token.colorInfo,
+                  }}
+                />
+              }
+              onClick={() => setManagingResourceLimit(row)}
+            />
+            <Button
+              type="text"
+              icon={
+                <AppstoreOutlined
+                  style={{
+                    color: token.colorInfo,
+                  }}
+                />
+              }
+              onClick={() => {
+                setManagingApp(row);
+              }}
+            />
+          </Flex>
+        </CellWrapper>
+      ),
+    },
+  ];
+
+  return (
+    <>
+      <Flex
+        direction="column"
+        align="stretch"
+        style={{
+          flex: 1,
+          ...style,
+        }}
+      >
+        <Flex justify="end" style={{ padding: token.paddingSM }}>
+          <Button
+            icon={<VerticalAlignBottomOutlined />}
+            style={{ backgroundColor: token.colorPrimary, color: 'white' }}
+            onClick={() => {
+              if (selectedRows.length === 0) {
+                message.warning(t('environment.NoImagesAreSelected'));
+                return;
+              }
+              if (selectedRows.some((image) => !image.installed)) {
+                setIsOpenInstallModal(true);
+                return;
+              }
+              message.warning(t('environment.AlreadyInstalledImage'));
+            }}
+          >
+            {t('environment.Install')}
+          </Button>
+        </Flex>
+        <Table<EnvironmentImage>
+          rowKey="id"
+          scroll={{
+            x: 1600,
+            y: window.innerHeight - 145 - 56 - 54,
```

Comment:
145: ref ) https://github.com/lablup/backend.ai-webui/pull/2615/files#diff-7513e7364d1bbef624ab929ba47fa51f2fcb5c0090f36db4f56e3837bb29bac5R78

56 : The space where the install button exists 

54: column title of table